### DISCUSSION TITLE
Et le gaz ?

### DISCUSSION THREAD

**Discussion by herveaurel** [saniho/apiEnedis#195] [General]
Bonjour à tous ! Je me demandais si il y avait aussi un moyen de récupérer les données du gaz qui sont sur le site Enedis comme pour l'électricité ?

**Comment by mdeweerd** [comment]
Pour le gaz j'utilise [homeassistant_gazpar_cl_sensor](/frtz13/homeassistant_gazpar_cl_sensor) .
Et pour l'eau (veolia) en IDF [mon adaptation de veolia_idf](/mdeweerd/veolia-idf) .

**Reply by herveaurel** [reply]
> Pour le gaz j'utilise [homeassistant_gazpar_cl_sensor](/frtz13/homeassistant_gazpar_cl_sensor) .
> 
> Et pour l'eau (veolia) en IDF [mon adaptation de veolia_idf](/mdeweerd/veolia-idf) .

Merci beaucoup. 
Je vois que pour le gaz il faut créer un compte sur le site ERDF, mais comme l'info est sur le site EDF chez moi, est-il nécessaire quand même de créer le compte chez ERDF ?

**Reply by mdeweerd** [reply]
On polue un peu apiEnedis avec des questions pour l'autre composant, mais bon.

Ce n'est pas chez ERDF (désormais Enedis) qu'il faut créer un compte mais chez GRDF.
La raison est simplement que le composant est fait pour le site de GRDF ce qui fonctionne pour tout le monde, alors que sinon il faudrait s'interfacer avec EDF, TotalEnergie, etc.
C'est tout comme apiEnedis qui cherche l'info à travers une gateway (et son API) qui utilise lui-même une API d'Enedis (anciennement ERDF).

**Reply by herveaurel** [reply]
> On polue un peu apiEnedis avec des questions pour l'autre composant, mais bon.
> 
> 
> 
> Ce n'est pas chez ERDF (désormais Enedis) qu'il faut créer un compte mais chez GRDF.
> 
> La raison est simplement que le composant est fait pour le site de GRDF ce qui fonctionne pour tout le monde, alors que sinon il faudrait s'interfacer avec EDF, TotalEnergie, etc.
> 
> C'est tout comme apiEnedis qui cherche l'info à travers une gateway (et son API) qui utilise lui-même une API d'Enedis (anciennement ERDF).

Merci. Oui tu as raison. 
A la base je pensais que c'était lié. 
Mais disons que ça peut servir à d'autres qui se poseraient la question et ça fait parti de l'entraide. 
Sur ce un grand merci pour toutes les infos !
